0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -38,6 +38,12 @@ Connectors *peer* with one another to exchange information used to determine the
 
 **Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
 
+### Interledger Networks
+
+The Interledger protocol stack can be used to create a network that effectively offers the same functionality as a centralized ledger; namely, the ability to safely transfer value from one account to another. But since an Interledger network is made up of autonomously run ledgers and connectors, without any central authority, these ledgers and connectors can then compete, at each hop of a given payment, to offer a good service at a low price. This will lead, through market economics, to cheaper transaction costs than those that can be expected on a centrally controlled system.
```

Comment:
Talking about how interoperability brings competition, which you would expect to drive down the rates, seems useful but I don't find the comparison to one centralized network very helpful. Right now there isn't one network. There are many networks. The key point is that all of those are disconnected and currently compete on reach.

Reply:
OK, I see now how that's confusing. I'm comparing Interledger to an *imaginary* centralized system which doesn't really exist.